auth-web/src/store/modules/business.ts

```diff
@@ -61,6 +61,31 @@ export default class BusinessModule extends VuexModule {
     await OrgService.createAffiliation(currentOrganization.id, requestBody)
   }
 
+  @Action({ rawError: true })
+  public async addNameRequest (requestBody: CreateNRAffiliationRequestBody) {
+    const currentOrganization: Organization = this.context.rootState.org.currentOrganization
+
+    // Create an affiliation between implicit org and requested business
+    return OrgService.createNRAffiliation(currentOrganization.id, requestBody)
+  }
+
+  @Action({ rawError: true })
+  public async updateFiling (filingBody: UpdateFilingBody) {
+    // Create an affiliation between implicit org and requested business
+    const updateResponse = await BusinessService.updateFiling(filingBody)
+    if (updateResponse?.status >= 200 && updateResponse?.status < 300) {
+      return updateResponse
+    } else {
+      // delete affiliation
```

Comment:
more comments will be helpful i think ; like why we have to delete the affiliation ? since the method name is updateFiling I was thinking it will handle on updatefiling logic.. but it has logic about deleting the affiliation[thats good ] ;but more comment will be really helpful

Reply:
will add more comments